lib/IO/String.pm

```diff
@@ -36,10 +36,48 @@ TODO
 
 #| An L<IO::Handle> implementation that writes to memory.
 class IO::String:ver<0.1.0>:auth<hoelzro> is IO::Handle {
-    has @.contents;
+    has Str $.buffer = '';
+    has Int $.pos = 0;
+
+    multi method open(IO::String:D:) {
+        my Str $new-buffer = '';
+        $!buffer := $new-buffer;
```

Comment:
Possibly. I need to play with it a bit to see if I understand how binding works. If we eliminate binding, then we don't need to worry about it.